project_iteration/models/project_task.py

```diff
@@ -1,24 +1,26 @@
-# -*- coding: utf-8 -*-
-# © 2018 Numigi (tm) and all its contributors (https://bit.ly/numigiens)
+# © 2019 Numigi (tm) and all its contributors (https://bit.ly/numigiens)
 # License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
 
 from odoo import api, fields, models
 
 
 class ProjectTaskWithParentProject(models.Model):
-    """Add parent_project_ids to tasks.
+    """Add parent_project_id to tasks.
 
     This field is used to navigate to the tasks from a parent project.
     """
 
     _inherit = 'project.task'
 
-    parent_project_ids = fields.Many2many(
-        'project.project', 'project_task_parent_rel', 'task_id', 'project_id',
-        string='Parent Project', compute='_compute_parent_project_ids', store=True)
+    parent_project_id = fields.Many2one(
```

Comment:
Don't we need a migration script for client using already this module ?

Reply:
Not for this. We are adding a new field and removing an old one.
These 2 cases are covered by a standard "odoo -u module".